docs/programme/area-owners.md

```diff
@@ -0,0 +1,151 @@
+# Area Owners
+
+In order to make sure that nothing falls through the cracks, and that work is
+not unduly piled onto a single person, we have a system of area owners. Ideally
+these area owners cover all parts of the programme. This list is not set in
+stone, and we expect it to change over time.
+
+It is wise to make sure that each area owner has a deputy, who can step in when
+they are unavailable, and can at least be aware of what is going on in their
+area.
+
+## Health and Safety
+
+Responsibilities:
+
+* Safety and compliance inspections
+* Risk assessments
+* Area wardens
+* Fire safety
+* Overnight safety at venues
+
+## Media / Press
+
+Responsibilities:
+
+* Photos
+* Organisation of group photo
+* Blog posts/announcements
+* Social media (incl. competitor social media)
+* Inviting guests
+* Arranging guest speaker
+* Swag
+* Competition booklet
+* Wristbands/lanyards
+* Reception
+* Media consent arrangements
+* Website (except srcomp)
+
+## Mentoring
+
+Responsibilities:
+
+* In schools
+* Helpdesk
+* First-line support
+* Roving help at competition
+* Microgames
+* Discord bot & moderation
+* Tech day content/structure
+* Power tools area
```

Comment:
Why did this end up under mentoring?

Reply:
Goooood question. I think because it ends up being more instruction than supervision?

Reply:
Where do you think it should live?

Reply:
Probably H&S as this is mainly about using tools safely.

Reply:
Will move

Reply:
c170a7961665a91ee83036394fd8da8dfa730578